Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -242,6 +242,14 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
                 try! self.handleDiff(payload: message.payload, baseURL: self.url)
             }
         }
+        channel.on("live_redirect") { [weak self] message in
```

Comment:
What event does `push_patch` trigger? I assume it's not `live_redirect` but maybe something like `live_patch`?

Same for `redirect`, I'm not sure if this PR will handle redirect or now.

Reply:
`push_patch` and `redirect` are separate events/replies. I can open separate PRs to handle those.

Reply:
Awesome. I'm not sure if I should "Approve" since I'm not really a core team member (...yet) but this looks great to me.

Thank you for solving my issue 🙏